### PR TITLE
aws client parameters

### LINKED ISSUE #40: geteventdata returns 502 because of missing parameters

### PR DESCRIPTION
**Pull request by atikenny**

### CODE REVIEW

**Comment by atikenny** [comment]
fixes #40

File: app/js/aws-client.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import apigClientFactory from 'aws-api-gateway-client';
 
 const awsClient = apigClientFactory.newClient({
-    invokeUrl: 'https://4ks5nf7ul7.execute-api.eu-central-1.amazonaws.com'
+    invokeUrl: 'https://4ks5nf7ul7.execute-api.eu-central-1.amazonaws.com',
+    region: 'eu-central-1'
```
Comment (csabbee) [inline app/js/aws-client.js:5]: isn't this coming from local settings?

Comment (atikenny) [reply app/js/aws-client.js:5]: The client in the browser needs this, not the aws cli.

Comment (csabbee) [reply app/js/aws-client.js:5]: Oh i see

**Review by csabbee — APPROVED** [review]

**Pull request merged**